### DISCUSSION TITLE
No screenpad brightness controls, camera button not functioning and more

### DISCUSSION THREAD

**Discussion by UX8402ZE** [seerge/g-helper#3279] [Q&A]
Hello I hope you're doing great!
I'm having trouble getting the dedicated camera button to work, the button used to switch the content of the main and screenpad is non functional and most important the hotkey Fn + Shift + F7 / F8 - Screenpad brightness Down / Up doesn't work for me. Current version is 0.196.0 on the Zenbook Pro 14 Duo UX8402ZE.

and final question: FN+F used to toggle performance modes but since FN+C/V /F4 is not supported on my device I can't really utilise them under extras, is there a workaround to get FN+F to be an option under extras?

**Comment by seerge** [comment]
@UX8402ZE hello, you need to [fill in issue template properly](https://github.com/seerge/g-helper/issues/new?assignees=&labels=&projects=&template=bug_report.yml) and attach all requested info, cause without logs i can't say much.

- But camera toggle, just changes camera permission in windows settings. And to work - app needs to be running as admin.
- `Fn + Shift + F7/F8` will work only if all asus services are stopped under `Extra` (or they handle hotkeys)

**Comment by UX8402ZE** [comment]
done I have posted the issue now.
